### PR TITLE
checksec: add new package

### PR DESCRIPTION
**Pull request by ja-pa**
Maintainer: me
Compile tested: Turris Omnia (TOS5), OpenWrt master, RPI3, OpenWrt 19.07
Run tested: Turris Omnia (TOS5), OpenWrt master, RPI3, OpenWrt 19.07

Description:
This PR adds a new package checksec.sh utility. It helps with checking properties of executables (like PIE, RELRO, PaX, Canaries, ASLR, Fortify Source).
The main reason why to make it as the package is the number of non-busybos utilities which it uses.

It could be useful in case of similar studies like this https://www.fkie.fraunhofer.de/content/dam/fkie/de/documents/HomeRouter/HomeRouterSecurity_2020_Bericht.pdf (section 3.3  Exploit Mitigation )

### CODE REVIEW

File: utils/checksec/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  CATEGORY:=Utilities
+  TITLE:=Utility to check PIE, RELRO, ASLR etc.
+  URL:=https://github.com/slimm609/checksec.sh
+  DEPENDS:=+bash
+endef
+
+define Package/checksec
+  $(call Package/checksec/default)
+  TITLE:=Utility to check PIE, RELRO, ASLR etc.
+  DEPENDS+=+file +binutils +procp-ps-sysctl +openssl-util +coreutils-stat
```
Comment (neheb) [inline utils/checksec/Makefile:35]: Needs additional dependencies on +procps-ng and +coreutils. +procp-ps-sysctl is not correct.

File: utils/checksec/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+endef
+
+define Package/checksec_automator
+  $(call Package/checksec/default)
+  TITLE:=Utility to use checksec for dirs
+  DEPENDS+=+checksec +coreutils-tee +findutils-find +grep
```
Comment (neheb) [inline utils/checksec/Makefile:41]: Needs an additional dependency on +coreutils .

File: utils/checksec/Makefile (hunk 1 above)
Comment (ja-pa) [reply utils/checksec/Makefile]: Thanks, fixed.

File: utils/checksec/Makefile (hunk 2 above)
Comment (ja-pa) [reply utils/checksec/Makefile:41]: I added coreutils dependency to checksec package

File: utils/checksec/Makefile [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/checksec/default
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Utility to check PIE, RELRO, ASLR etc.
+  URL:=https://github.com/slimm609/checksec.sh
+  DEPENDS:=+bash
```
Comment (neheb) [inline utils/checksec/Makefile:29]: Remove and place in checksec. checksec_automator will inherit it.

Comment (ja-pa) [reply utils/checksec/Makefile]: Removed from checksec/default (also with TITLE)

**Pull request merged**